api/src/wfl/service/datarepo.clj

```diff
@@ -242,35 +277,18 @@
    :name        (str name "_" table)
    :profileId   defaultProfileId})
 
-;; HACK: (str "datarepo_" name) is a hack while accessInformation is nil.
```

Comment:
Fun to delete!

Reply:
Should have lead with, thanks @tbl3rd for prodding me to look upstream to see if we could streamline this.

Reply:
Thank you for following up. I just voiced a hunch. You made it happen.